translations/messages.en.yaml

```
@@ -11,3 +11,4 @@ sylius_price_history:
         days: 'Days'
         price_history: 'Price history'
         show_history_for_channel_pricing: 'Show history for channel pricing'
+        the_lowest_price_days_before_the_discount_was: 'The lowest price %days% days before the discount was %price%'
```

```suggestion
        lowest_price_days_before_the_discount: 'The lowest price of %days% days before the discount: %price%'
```
I would rather have something like that, but I don't have a strong opinion